### PR TITLE
BZ-1979431: Adding link to configuring project node selectors

### PR DESCRIPTION
**Pull request by bergerhoffer**
Preview: https://deploy-preview-35894--osdocs.netlify.app/openshift-enterprise/latest/post_installation_configuration/cluster-tasks.html#creating-an-infra-node_post-install-cluster-tasks

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@kasturinarra Can you please review this? It's a followup to #34928, which you had previously reviewed.

Preview: https://deploy-preview-35894--osdocs.netlify.app/openshift-enterprise/latest/post_installation_configuration/cluster-tasks.html#creating-an-infra-node_post-install-cluster-tasks

**Review by codyhoag — APPROVED** [review]
LGTM!

File: post_installation_configuration/cluster-tasks.adoc [hunk 1]
```diff
@@ -513,6 +513,10 @@ include::modules/machineset-creating.adoc[leveloffset=+2]
 
 include::modules/creating-an-infra-node.adoc[leveloffset=+2]
 
+.Additional resources
+
+* For information on how to configure project node selectors to avoid cluster-wise node selector key conflicts, see xref:../nodes/scheduling/nodes-scheduler-node-selectors.adoc#project-node-selectors_nodes-scheduler-node-selectors[Project node selectors].
```
Comment (kasturinarra) [inline post_installation_configuration/cluster-tasks.adoc:518]: @bergerhoffer a small typo replace cluster-wise / cluster-wide

**Comment by kasturinarra** [comment]
@bergerhoffer a small typo, otherwise looks good to me, thanks !!

**Comment by bergerhoffer** [comment]
Good catch thanks @kasturinarra!

**Pull request merged**